mods/gen2/formats-data.js

```diff
@@ -179,21 +176,22 @@ exports.BattleFormatsData = {
 		},
 		tier: "NFE",
 	},
-	blastoise: {
-		randomSet1: {
+	blastoise: {//done
```

Comment:
Looks great, I think I might expand it to three sets for more accuracy though (e.g. stuff like stick to lefties for stalk, don't have 3 attacks without ibeam)